src/covid_shared/cli_tools/logging.py

```diff
@@ -1,40 +1,222 @@
+import logging
 from pathlib import Path
 import sys
-import typing
-from typing import Union
+from typing import TextIO, Tuple, Union
 
 from loguru import logger
 
 from covid_shared import paths
 from covid_shared.shell_tools import mkdir
 
 
-DEFAULT_LOG_MESSAGING_FORMAT = ('<green>{time:YYYY-MM-DD HH:mm:ss.SSS}</green> | '
-                                '<cyan>{name}</cyan>:<cyan>{function}</cyan>:<cyan>{line}</cyan> '
-                                '- <level>{message}</level> - {extra}')
-
+JOBMON_LOGGING_LEVEL = 5  # A lower level than logging.DEBUG.
+DEFAULT_LOG_MESSAGING_FORMAT = (
+    '<green>{time:YYYY-MM-DD HH:mm:ss.SSS}</green> | '
+    '<cyan>{name}</cyan>:<cyan>{function}</cyan>:<cyan>{line}</cyan> '
+    '- <level>{message}</level> - {extra}'
+)
 LOG_FORMATS = {
     # Keys are verbosity.  Specify special log formats here.
-    0: ("ERROR", DEFAULT_LOG_MESSAGING_FORMAT),
+    0: ("WARNING", DEFAULT_LOG_MESSAGING_FORMAT),
```

Comment:
What happened to "ERROR"?

Reply:
Nothing.  We just log warnings too now, which should have been the default for the lowest verbosity setting.